.gitmodules

```diff
@@ -17,7 +17,7 @@
 [submodule "src/interfaces"]
 	path = src/interfaces
 	url = https://github.com/THI-Drone/interfaces.git
-	branch = dev
+	branch = feat-fcc-bridge-defs 
```

Comment:
Is this on purpose?

Reply:
No, that's something that got introduced by a merge with the other branch

Reply:
Fixed in https://github.com/THI-Drone/thi-drone-ws/pull/20/commits/a9b0ab32a7a65e40b82d5a2a8f422f351f3be836